src/components/reports/ReportForm.tsx

```diff
@@ -132,10 +161,9 @@ const ReportForm = () => {
           <FieldSet name="result.reportingScientistTitle" label="Reporting scientist title"/>
           <br/>
           <button type="submit">Submit</button>
-
-          {result !== "" && <textarea id="resultOutput" role="alert" rows={20} defaultValue={result}/>}
         </Form>
       </Formik>
+      {result !== "" && <textarea id="resultOutput" role="alert" rows={20} defaultValue={result}/>}
```

Comment:
What is the purpose of rendering the result here?

Reply:
mostly for debugging and demo to show what is being sent to the FHIR API. Will be removed in the final application